projects/graph/src/graph_demo.py

```diff
@@ -3,14 +3,98 @@
 """
 Demonstration of Graph and BokehGraph functionality.
 """
-
+import random 
+from graph import Graph
+from draw import BokehGraph
 from sys import argv
 
 
-def main():
-    pass  # TODO
+def create_default_graph():
+    graph = Graph()  # Instantiate your graph
+    graph.add_vertex('5')
```

Comment:
You should go back and try to see if you can refactor this to use a more programmatic approach to do the same thing? Can you do it in fewer lines?